WordPress/Classes/ViewRelated/System/Notices/NoticeView.swift

```diff
@@ -260,7 +298,7 @@ class NoticeView: UIView {
     private enum Metrics {
         static let cornerRadius: CGFloat = 4.0
         static let dualLayoutMargins = UIEdgeInsets(top: 6.0, left: 6.0, bottom: 6.0, right: 6.0)
-        static let labelLineSpacing: CGFloat = 18.0
```

Comment:
needed to update this value after removing: `labelStackView.isBaselineRelativeArrangement = true`